RELEASE_NOTES.txt

```diff
@@ -1,6 +1,120 @@
 Apache Ignite Release Notes
 ===========================
 
+Apache Ignite In-Memory Distributed Database 2.14.0
+-----------------------------------------------------------
+
+(!) WARNINGS:
+* Removed LOCAL caches support.
+* Removed scalar module.
+
+Ignite:
+* Added CDC binary meta change events.
+* Added a snapshot create operation metrics.
+* Added a user info to the sql queries view.
+* Added cache change events for CDCCosumer.
+* Added cache metrics command for Control Script.
+* Added control.sh command that schedules index rebuild via the maintenance mode.
+* Added opportunity to specify single partition for IndexQuery.
+* Added pluggable affinity mapping function for the partition awareness usage.
+* Added snapshot status command to control.sh and JMX.
+* Added support of optional beans for SpringResource annotation injection.
+* Added system view and histogram to show the distribution of hot and cold pages in the page memory.
+* Added the ability to define a custom path for the snapshot operation.
+* Added the ability to intercept calls to service methods.
+* Consistency check now able to fix counters after the consistency fix.
+* Consistency check now support cacheGroup as a param as well.
+* Fixed 'Invalid cross-device link error' when extending WAL segments size.
+* Fixed ClassCastException that appears during deployment manager stop in Java versions 11.0.16+ and 17.0.4+.
+* Fixed Integer value is out of bounds when the dataregion maxSize is large enough.
+* Fixed an issue that caused a node is trying to resolve its hostname if IP is configured with IGNITE_LOCAL_HOST.
+* Fixed an issue that caused a node resolves obsolete addresses from the previously restarted and killed nodes.
+* Fixed an issue that caused inconsistent keys after deletion during rebalance.
+* Fixed an issue that could lead to a block of the management pool in case of using clearSync.
+* Fixed an issue that could lead to unexpected partition map exchange on client nodes.
+* Fixed an issue where the snapshot error was not propagated to the initiating node.
+* Fixed an issue with potential node failure during cancelling of compute jobs with cache operations.
+* Fixed configured event listeners resource injection order.
+* Fixed durable background task start on BLT node joined to the active cluster.
+* Fixed expiration doesn't apply for cache entries replicated with CDC.
+* Fixed incorrect params of GridCacheManager#stop method.
+* Fixed overflowing async operation permits maximum for explicit transaction with single write entry.
+* Fixed partition update counter tracking on primary after the node restart.
```

Comment:
probably, this was meant:
42. Fixed partition update counter tracking on primary **server** after the node restart.